app/src/main/assets/native-10.6/exoplayer.js

```diff
@@ -303,7 +331,7 @@ define(['events', 'appSettings', 'loading', 'playbackManager'], function (events
                             var videoCodec = codecs.videoCodecs[index];
                             videoCodecs.push(videoCodec.codec);
 
-                            var profiles = videoCodec.profiles.join('|');
+                            var profiles = videoCodec.profiles.reverse().join('|');
```

Comment:
JF 10.7 can transcoding to `H264 High` and `HEVC Main`. Remuxing to `H264 High 10` and `HEVC Main 10` is also supported. I think exoplayer can handle HEVC content over TS and fMP4.

For maximum compatibility, if `H264 High 10` or `HEVC Main 10` profiles are available on the device, please put them at the end.

Reply:
I don't know about `HEVC` or in this case `H264 High 10` since my device is not compatible as you can see in the example, but in theory there should be no compatibility problems, since the list of codecs is generated according to the device. That is precisely why I inverted the list to take the best possible profile and achieve better quality transcodes.

See

- https://github.com/jellyfin/jellyfin/blob/cb94b5a94460be58e1b90366ef0ee45b8de1c0d1/Emby.Dlna/Profiles/XboxOneProfile.cs#L33
- https://github.com/jellyfin/jellyfin-web/blob/76d2b7018cf1b8f30525d6dde59e8a4210ecc48c/src/scripts/browserDeviceProfile.js#L737
https://github.com/jellyfin/jellyfin-android/blob/6cc40f6571f554859446ad35cb4e58138a0d492d/app/src/main/assets/native-10.6/nativeshell.js#L115-L119

Reply:
Seems we haven't enable HEVC transcoding to Android client. That's fine for now.